### PR TITLE
fix: Add visibility to box for viewing menu items

### PR DESCRIPTION
**Pull request by maloun96**
### SUMMARY
Add visibility to filter box for viewing menu items

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE
image
AFTER
image

### TEST PLAN
Go to dashboard
Set window size small that 3 dots menu can not fit into the box
See that menu is not visible at all.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API
- [x] Fix: https://github.com/apache/incubator-superset/issues/12108

### CODE REVIEW

**Comment by rusackas** [comment]
@kkucharc This changes something you touched just a couple weeks ago... could you quickly verify this doesn't break the changes you made? I think it was around the Filter Box "Apply" scrolling.

**Comment by junlincc** [comment]
Screen Shot 2021-01-01 at 4 22 43 PM
thanks for the fix! 
i can't fully test it due to this error, not sure rebasing will help? not seeing this error elsewhere @maloun96

**Comment by villebro** [comment]
> Screen Shot 2021-01-01 at 4 22 43 PM
> 
> thanks for the fix! i can't fully test it due to this error, not sure rebasing will help?
 
@junlincc do you have `GLOBAL_ASYNC_QUERIES` feature flag turned on?

File: superset-frontend/src/dashboard/stylesheets/components/chart.less [hunk 1]
```diff
@@ -23,7 +23,7 @@
   background-color: @lightest;
   position: relative;
   padding: 16px;
-  overflow-y: auto;
+  overflow-y: visible;
```
Comment (kkucharc) [inline superset-frontend/src/dashboard/stylesheets/components/chart.less:26]: Unfortunately this change will affect `Region Filter`:
`overflow-y: visible` (now):
Screenshot 2021-01-04 at 16 15 00

and  `overflow-y: auto` (was):
Screenshot 2021-01-04 at 16 14 46

I changed scroll solution back [here](https://github.com/apache/incubator-superset/issues/11419) but I believe there must be some better solution here.

**Review by kkucharc — COMMENTED** [review]
I just added my comment, but maybe it's better to solve this issue and find another solution for Region Filter.

File: superset-frontend/src/dashboard/stylesheets/components/chart.less (hunk 1 above)
Comment (maloun96) [reply superset-frontend/src/dashboard/stylesheets/components/chart.less:26]: Kapture 2021-01-04 at 19 20 42
@kkucharc @junlincc

**Review by junlincc — COMMENTED** [review]
This PR fixes the original issue but it seems introduce 2 critical issues below.
@maloun96 please take another look, thanks!
1. can not resize filter box individually, it drags the chart near by as resizing 

https://user-images.githubusercontent.com/67837651/103612520-7869c700-4ed9-11eb-9b09-7633a5a14cab.mov

2. all other charts(not filter box)can not be resized at all.

https://user-images.githubusercontent.com/67837651/103612553-8c152d80-4ed9-11eb-8164-9171d386cfaa.mov

**Review by rusackas — APPROVED** [review]

File: superset-frontend/src/dashboard/stylesheets/components/chart.less (hunk 1 above)
Comment (rusackas) [reply superset-frontend/src/dashboard/stylesheets/components/chart.less:26]: @kkucharc I just pulled this, and confirm that what I see looks like @maloun96 's video above. While the scrolling is... weird, it seems comparable to what was there before. I think we're OK, though I hope to find a better design solution to tiny charts/filters like this, as the scrolling is goofy. 

I think we're OK here, but let me know if you see any remaining issue.

**Comment by rusackas** [comment]
> This PR fixes the original issue but it seems introduce 2 critical issues below.
> @maloun96 please take another look, thanks!
> 
> 1. can not resize filter box individually, it drags the chart near by as resizing
> 
>  Screen.Recording.2021-01-04.at.10.07.25.PM.mov 
> 1. all other charts(not filter box)can not be resized at all.
> 
>  Screen.Recording.2021-01-04.at.10.07.41.PM.mov

I believe issue #1 is due to the two charts being part of a "Column" and you're resizing that column. If I take them out of the column, they resize individually just fine.

I also see #2, but I also see it on `master`. Interestingly, the resize cursor on the right edge doesn't show, but if you try the bottom right hand corner, you'll see the diagonal resize cursor, and it works just fine.

Therefore, I'm approving this PR, but will hold off from merging without your thumbs up.

**Comment by junlincc** [comment]
@rusackas you are right about issue 1, my mistake!
i can also confirm issue 2 is on master. let's address it in a separate PR

**Review by junlincc — APPROVED** [review]

**Pull request merged**